api/src/test_k6_rate_limiter.js

```diff
@@ -0,0 +1,58 @@
+import { sleep, group } from "k6";
+import http from "k6/http";
+
+export const options = {
+  thresholds: {},
+  scenarios: {
+    Scenario_1: {
+      executor: "ramping-vus",
+      gracefulStop: "30s",
+      stages: [{ target: 100, duration: "1m" }],
+      gracefulRampDown: "30s",
+      exec: "scenario_1",
+    },
+  },
+};
+
+export function scenario_1() {
+  let response;
+  const youngs = [
+    { young: "63872bab3c891a1e1f28c45f" },
+    { young: "64d2177b8dc4859809e75b2d" },
+    { young: "64d218aa8dc4859809e75b2e" },
+    { young: "64d218bb8dc4859809e75b2f" },
+    { young: "64d218ec8dc4859809e75b30" },
```

Comment:
I haven't started to look into the tests yet, but why is these IDs hard coded?

Reply:
It's for the loadtesting of the ratelimiter , this are id from users from the stagging that i created for the purpose of the test